src/tests/ttls/CMakeLists.txt

```
@@ -0,0 +1,17 @@
+add_custom_command(
+  OUTPUT test_t.c
+  DEPENDS ../test.edl
+  COMMAND openenclave::oeedger8r --trusted
+          ${CMAKE_CURRENT_SOURCE_DIR}/../test.edl ${DEFINE_OE_SGX})
+
+add_enclave_library(erttest_ttls_lib OBJECT enc.cpp test_t.c)
+enclave_include_directories(
+  erttest_ttls_lib PRIVATE ${CMAKE_CURRENT_BINARY_DIR}
+  ${CMAKE_SOURCE_DIR}/../3rdparty/ttls/include)
+enclave_link_libraries(erttest_ttls_lib PRIVATE oe_includes)
```

```suggestion
enclave_include_directories(erttest_ttls_lib PRIVATE
                            ${CMAKE_CURRENT_BINARY_DIR})
enclave_link_libraries(erttest_ttls_lib PRIVATE oe_includes ttls)
```
let's inherit the include dir